<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>853</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hash Generator v0.3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>841</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Hashes</string>
    </property>
    <widget class="QPlainTextEdit" name="fieldSHA256">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btClipboardSHA256">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btClipboardSHA512">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="fieldSHA512">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>190</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="fieldMd5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btClipboardMD5">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkMD5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>94</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>MD5</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkSHA256">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>94</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA256</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkSHA512">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>94</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SHA512</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="btGenerate">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btClear">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilepath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Arquivo:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="fieldFile">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>20</y>
      <width>456</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Checksum  </string>
    </property>
    <widget class="QPlainTextEdit" name="fieldChecksum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>671</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btChecksum">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>410</y>
      <width>861</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="acSave"/>
    <addaction name="acSaveAs"/>
    <addaction name="separator"/>
    <addaction name="acQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Sobre">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Sobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="acSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="acQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="acGenerateHash">
   <property name="text">
    <string>Generate hash</string>
   </property>
  </action>
  <action name="acClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="acClipboard_MD5">
   <property name="text">
    <string>Clipboard MD5</string>
   </property>
  </action>
  <action name="acClipboard_SHA256">
   <property name="text">
    <string>Copy Clipboard SHA256</string>
   </property>
   <property name="toolTip">
    <string>Clipboard SHA256</string>
   </property>
  </action>
  <action name="acClipboard_SHA512">
   <property name="text">
    <string>Copy Clipboard SHA512</string>
   </property>
   <property name="toolTip">
    <string>Clipboard SHA512</string>
   </property>
  </action>
  <action name="acRegenerate">
   <property name="text">
    <string>Regenerate Hashes</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheckMD5">
   <property name="text">
    <string>MD5</string>
   </property>
  </action>
  <action name="actionCheckSHA256">
   <property name="text">
    <string>SHA256</string>
   </property>
  </action>
  <action name="actionCheck512">
   <property name="text">
    <string>SHA512</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
